src/Task/Git/CommitMessage.php

```diff
@@ -164,6 +164,12 @@ private function enforceTextWidth(ContextInterface $context)
 
         $errors = [];
         $lines = preg_split('/\R/u', $commitMessage);
+        foreach ($lines as $index => $line) {
```

Comment:
I also think this removal should not be done inside the `enforceTextWidth()` method. It might also break other checks like e.g. `subjectHasTrailingPeriod()`. It's probably best to move this normalisation to a separate method and run that method here:
https://github.com/Landerstraeten/grumphp/blob/89ca2b71fe6073346ca00fec09109a1727795b7e/src/Task/Git/CommitMessage.php#L101